### DISCUSSION TITLE
Nomes de branchs e ramificações

### DISCUSSION THREAD

**Discussion by vitorcbs** [techexpertspro/portal-ascepa#21] [Ideas]
Eae pessoal, beleza?
Levantei um ponto durante a aula 02 a cerca do nomes de branchs e ramificações de features, fixs, refactor, etc. Pensando na melhor identificação do conteudo das branchs, alterações, localização de erros e outros benefícios.
Para quem quiser saber mais sobre, deixo esse [artigo do medium](https://medium.com/@jacoblarte/padr%C3%B5es-de-nomenclatura-para-branches-e-commits-dfe10c9392bd) para consulta.

**Comment by FelipeDuarteLuna** [comment]
Bom dia Vitorcbs,

Vê que você criou o arquivo de CONTRIBUTING.md, ficou bacana. Mas acho que podemos melhorar, ditando as regras de como queremos os commit's, pull request , versionamento etc... Exemplo de projetos onde é feito esse processo:

https://github.com/po-ui/po-angular/blob/master/CONTRIBUTING.md

Eu fiz isso há alguns anos atrás, em um projeto . Ignore os emoticons, mas é uma maneira de padronizar, na época não manjava muito e copiei de uns cara que manjava mais que eu.

https://github.com/FelipeDuarteLuna/Portal-NFS-e/wiki

**Reply by vitorcbs** [reply]
Opa Felipe,
Vou dar uma olhada nesse exemplo e incrementar o contributing.md.
